srcpkgs/vpnd/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'vpnd'
+pkgname=vpnd
+version=0.1.1
+revision=1
+archs=noarch
+depends="bash yad"
+short_desc="Systray notifier for Void Linux packages"
+maintainer="Benjamin Slade <slade@lambda-y.net>"
+license="GPL-3.0-or-later"
+homepage="https://gitlab.com/emacsomancer/vpnd"
+distfiles="https://gitlab.com/emacsomancer/vpnd/-/archive/${version}/${pkgname}-${version}.tar.gz"
+checksum=49e148766f1e0bdb4a48d58bd72448e72c977f4c326393d694ae0cce2b42f6f9
+
+do_install() {
+	vbin vpnd
+	vmkdir usr/share/icons/vpnd
+	vcopy *.png usr/share/icons/vpnd
+	vmkdir etc
+	vcopy vpnd.conf etc
```

Comment:
there is vconf

Reply:
fixed in https://github.com/void-linux/void-packages/commit/4a686b34bf2eeaf74359eaa334a040f7b529d1dd